.github/workflows/ci.yml

```diff
@@ -7,8 +7,10 @@ name: CI
 on:
   push:
     branches: [main]
+    paths: ['src/**', 'tests/**']
   pull_request:
     branches: [main]
+    paths: ['src/**', 'tests/**']
```

Comment:
ooh I didn’t know this was possible. Hmm now we could bring over the `docs` into this repo 🤔